prog-o-meter.py

```diff
@@ -143,23 +150,30 @@ def add_day(self):
         self.days_remaining = self.GOAL - self.days
         self.completion_date = self.get_completion_date(self.days_remaining)
         self.canvas.itemconfig(self.rectangle_list[self.days-1], fill = "green")
+        self.canvas.itemconfig(self.encourage_text, text = "".join(self.encourage[self.new_no(self.x)]))
         update_days_file(self.filename, self.days)
         self.canvas.itemconfig(self.countdown_text, text = "".join(("You have ", str(self.days_remaining), " days left!\n\n", "If you code everyday, you will be done with this project on ", self.completion_date)))
-        if self.days >=self.GOAL:        # Disable add_day_button if goal have been reached 
-            self.add_day_button.config(state = "disabled")             
-
+        if self.days >=self.GOAL:        # Disable add_day_button if goal have been reached
+            self.add_day_button.config(state = "disabled")
+            self.canvas.itemconfig(self.hog, text=("".join(("Congrats! ", self.username))))
+    def new_no(self, x):

```

Comment:
When we write new functions, we need to document them with a docstring. A docstring is a comment that describes what the function do, what it takes in, and what it returns (you can read more about it [here](https://www.python.org/dev/peps/pep-0257/) if you are inbterested. 
For your docstring you need:

"""ONE_LINE_DESCRIPTION_OF_FUNCTION 
[BLANK LINE HERE]
(if necessary you can give a longer explanation over multiple lines here)
Args:
    x: here_you_describe_what_x_is (and also, it would be best if you changed the name of X, so the name itself said 
    more about what the argument is)
Returns:
    here_you_describe_what_it_returns

If you want, check out the code for inspiration of how other functions are documented. Not all function takes in arguments, so not all docstrings will have an args: list. Not all functions return something, so not all docstrings will have a returns: list.